apps/condo/domains/meter/schema/MeterReportingPeriod.js

```diff
@@ -66,46 +66,37 @@ const MeterReportingPeriod = new GQLListSchema('MeterReportingPeriod', {
             kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
         },
 
-        start: {
+        startAt: {
```

Comment:
Naming format corresponds to timestamp, rather than integer.
Maybe `startDayOfMonth` and `finishDayMonth`?

Reply:
This name has already been approved at the architecture meeting

Reply:
Approvement diverges from our implicit standard. Up to you to decide what to do.